solutions/remotemonitoring/armtemplates/basic-static-map.json

```diff
@@ -385,6 +385,14 @@
             "metadata": {
                 "description": "The certficate key that needs to be updated to the VM"
             }
+        },
+        "bingMapsLocation": {
+            "comments": "Marked for deletion, unused",
```

Comment:
should these comments be in metadata or should the other ones be out of the metadata section?